sources/packages/backend/apps/api/src/utilities/upload-utils.ts

```diff
@@ -76,6 +76,20 @@ const fileFilter = (
   }
 };
 
+/**
+ * TXT file filter, used by multer (node.js middleware),
+ * to allow only the specific MIME types files.
+ * @param file received file.
+ * @param callback callback to return the result of the validation.
+ */
+export const textFileFilter = (
```

Comment:
mm, I see the same logic for CSVFileFilter too, can't refuse the same methods by doing some adjustments like, 
![image](https://github.com/bcgov/SIMS/assets/77353155/965bd054-ca44-40c1-9c87-c7fe07545cd0)

Reply:
Had a call and resolved, its better to have it separated.